edgedb-protocol/src/serialization/decode/queryable.rs

```diff
@@ -0,0 +1,3 @@
+mod scalars;
+mod tuples;
+mod collections;
```

Comment:
Another nit: all text files should end with an empty newline. @tailhook we need to add an editor-config to our rust repos.

Reply:
I'd go one step further and add a `.rustfmt.toml` so the auto-formatter can take care of all that stuff.

Reply:
Well, I'm not into auto-formatters. They are occasionally actively make code worse. At other times they are unnecessary barrier for contributions. And they never improve the code. So I personally prefer to accept occasional long lines instead.

Editor-config is fine though. Feel free to open PR.